src/Microsoft.ML.CpuMath/IntUtils.cs

```diff
@@ -285,7 +285,7 @@ private static int CbitHighZero(ulong u)
         /// returning the quotient and placing the remainder in <paramref name="rem"/>. Throws on overflow.
         /// </summary>
 #if !CORECLR
-        [DllImport(Thunk.NativePath), SuppressUnmanagedCodeSecurity]
+        [DllImport(Thunk.NativePath, CallingConvention =CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Comment:
```suggestion
        [DllImport(Thunk.NativePath, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
```

Reply:
I reverted this change

---
In reply to: [239656605](https://github.com/dotnet/machinelearning/pull/1833#discussion_r239656605) [](ancestors = 239656605)